### PR TITLE
Test more XR things

### PR DESCRIPTION
**Pull request by Manishearth**
This tests all the things that are currently possible and useful to test.

I'm going to avoid expanding rust-webvr's mock functionality for now, instead growing it in rust-webxr

Things that we support which aren't tested yet:

 - viewport values (there's no way in the xrtest api to mock these yet)
 - input (no support in rust-webvr for mocking)
 - floor spaces (no way in the xrtest api to mock these yet)

r? @asajeffrey

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23557)

### CODE REVIEW

File: tests/wpt/mozilla/meta/MANIFEST.json [hunk 1]
```diff
@@ -21133,11 +21154,27 @@
    "testharness"
   ],
   "webxr/obtain_frame.html": [
+   "5182093b33f0870550157d648e41376a91fd95ce",
+   "testharness"
+  ],
+  "webxr/obtain_frame_BACKUP_17524.html": [
```
Comment (Manishearth) [inline tests/wpt/mozilla/meta/MANIFEST.json]: argh

**Comment by asajeffrey** [comment]
r=me after appropriate squashing

**Comment by Manishearth** [comment]
Once rust-webxr is mature enough, I'll add mocking hooks to it that include hooks for the currently-untested stuff.

**Comment by CYBAI** [comment]
Travis complains about tidy 👀

```
Running the WPT lint on ./tests/wpt/mozilla/tests/...
Checking files for tidiness...
./tests/wpt/mozilla/tests/webxr/obtain_frame.html:24: trailing whitespace
```

**Comment by CYBAI** [comment]
@bors-servo retry

- #23290

**Pull request merged**